### PR TITLE
Fix some typos in error messages

### PR DESCRIPTION
**Pull request by mattj-io**

### CODE REVIEW

File: lib/proxy/dhcp/subnet.rb [hunk 1]
```diff
@@ -187,7 +187,7 @@ def valid_range args = {}
         raise Proxy::DHCP::Error, "Range does not belong to provided subnet" unless self.include?(from) and self.include?(to)
         from = IPAddr.new(from)
         to   = IPAddr.new(to)
-        raise Proxy::DHCP::Error, "#{from} can't be lower IP adderss then #{to} - chage the order?" if from > to
+        raise Proxy::DHCP::Error, "#{from} can't be lower IP address then #{to} - change the order?" if from > to
```
Comment (domcleal) [inline lib/proxy/dhcp/subnet.rb]: "than" too

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
:+1: @lzap squash and merge this at the same time as the other please :)

**Comment by lzap** [comment]
Squashed and pushed together with https://github.com/theforeman/smart-proxy/pull/148

Thanks!

**Pull request closed without merging**